assets/training/vision/environments/automl-dnn-vision-gpu/context/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN conda env create -p $AZUREML_CONDA_ENVIRONMENT_PATH -f conda_dependencies.ya
     conda clean -a -y
 
 # vulnearbility fix
-RUN pip install pyarrow==14.0.1
+RUN pip install pyarrow==14.0.2
```

Comment:
nit: Wondering 14.0.2 specifically? In other envs, we have 14.0.1 though.

Reply:
From envs/Latest/AzureML-AutoML-DNN-Vision-GPU/context/Dockerfile